Intersect.Server/Database/Item.cs

```diff
@@ -20,44 +21,24 @@ public Item()
         {
         }
 
-        public Item(Guid itemId, int quantity, bool incStatBuffs = true) : this(
-            itemId, quantity, null, null, incStatBuffs
+        public Item(Guid itemId, int quantity, ItemProperties ItemProperties = null) : this(
```

Comment:
The parameter name should just be `properties`, we already know they're item properties.

Reply:
This constructor is also _not_ functionally equivalent anymore.

Reply:
I would re-create the old constructor with the boolean renamed to `randomizeStatModifiers` and this constructor would be a _new_ constructor and `properties` should _not_ have a default-null value.